### PR TITLE
[IMP] website, *: simplify kanban archs

### PR DESCRIPTION
**Pull request by omra-odoo**
*website_blog,website_crm,website_event_exhibitor,website_event_track,
  website_forum,website_livechat,website_sale,website_slides,website_sms
In this commit we have simplified the kanban arch for the website and their related modules.the goal is to simplify them, make them easier to read and use bootstrap utility
classnames.

- Previously, we used kanban-box, but now we are using kanban-card instead.
- Deprecated oe_kanban_global_click and oe_kanban_global_click_edit.
- More use of `<field/>` tags
- Removed the oe_kanban_colorpicker class and replaced it with the kanban_color_picker widget.
- Changed type='edit' to type='open' to open records. since version 16, records always open in edit mode by default.
- kanban_image from rendering context, is deprecated so we use `<field name="..." widget="image"/>` instead
- kanban_color, kanban_getcolor and kanban_getcolorname are deprecated use new attribute highlight_color="color_field_name" on root node

Task-3992107

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_livechat/views/website_visitor_views.xml [hunk 1]
```diff
@@ -60,13 +60,11 @@
...
                 </div>
                 <div t-if="record.livechat_operator_id.raw_value" class="col-lg col-sm-4 col-6 py-0 my-2">
-                    <div>
-                        <img name="livechat_operator_id"
-                            class="oe_avatar rounded-circle" width="19" height="19"
-                            t-attf-src="/web/image/res.partner/#{record.livechat_operator_id.raw_value}/avatar_128"
-                            alt="Operator Avatar"/>
-                        <b><field name="livechat_operator_name"/></b>
-                    </div>
+                    <img name="livechat_operator_id"
```
Comment (aab-odoo) [inline addons/website_livechat/views/website_visitor_views.xml:63]: why not using widget image?

File: addons/website_sale/views/product_image_views.xml [hunk 2]
```diff
@@ -41,20 +41,15 @@
...
-                <field name="image_1920"/>
                 <field name="sequence" widget="handle"/>
                 <templates>
-                    <t t-name="kanban-box">
-                        <div class="card oe_kanban_global_click p-0">
-                            <div class="o_squared_image">
-                                <img class="card-img-top" t-att-src="kanban_image('product.image', 'image_1920', record.id.raw_value)" t-att-alt="record.name.value"/>
+                    <t t-name="kanban-card" class="p-0 border-0">
+                        <div class="card">
+                            <div class="o_squared_image" t-att-alt="record.name.value">
```
Comment (aab-odoo) [inline addons/website_sale/views/product_image_views.xml:48]: t-att-alt on field, not on div

File: addons/website_slides/views/slide_slide_views.xml [hunk 3]
```diff
@@ -203,67 +203,52 @@
...
-                                        <span>
-                                            <i class="fa fa-eye me-2" aria-label="Views" role="img" title="Views"/><field name="total_views"/>
-                                        </span>
-                                        <field name="user_id" widget="many2one_avatar_user"/>
+                        <t t-name="kanban-card" class="flex-row">
+                            <aside class="o_kanban_aside_full">
+                                <t t-if="record.image_128.raw_value">
+                                    <div class="o_kanban_image_fill position-relative w-100">
+                                        <field name="image_128" t-att-alt="record.id.value" class="h-100" widget="image" options="{'img_class': 'object-fit-cover'}"/>
+                                        <field name="channel_id" class="position-absolute bottom-0 end-0 bg-light" widget="image" options="{'preview_image': 'image_128', 'img_class': 'object-fit-contain'}" style="box-shadow: -1px -1px 0 1px white; width: 40%;"/>
```
Comment (aab-odoo) [inline addons/website_slides/views/slide_slide_views.xml:212]: I would avoid inline style and define it in scss instead. Where does this style come from by the way?

Comment (omra-odoo) [reply addons/website_slides/views/slide_slide_views.xml]: i have added new class `o_inner_image`, the style comes from [o_kanban_image_inner_pic](https://github.com/odoo/odoo/blob/0b8b14c5ae5ff06ce8914a94b602a7fac2c2655b/addons/web/static/src/views/kanban/kanban_record_legacy.scss#L137) here

File: addons/website/static/src/scss/website.backend.scss [hunk 4]
```diff
@@ -108,7 +108,9 @@
         margin-left: $grid-gutter-width * 2;
         margin-top: 16px;
         margin-bottom: 16px;
-
```
[the remark this answers is not in the document]
Comment (omra-odoo) [reply addons/website/static/src/scss/website.backend.scss:120]: i have added the scss of o_theme_preview_kanban_view near the `theme_preview_kanban.js`

File: addons/website_slides/static/src/scss/slide_views.scss [hunk 5]
```diff
@@ -11,6 +11,11 @@
     }
 }
 
+.o_inner_image {
```
Comment (aab-odoo) [inline addons/website_slides/static/src/scss/slide_views.scss:14]: this classname is generic (not prefixed by any module name), it looks like a framework classname => rename it to o_website_slides_inner_image

Also, the selector is too weak. As soon as this addon is installed, any element in any addon with classname `o_inner_image` will come with those css rules. You want to scope these rule to the kanban view where it is used.

File: addons/website_slides/views/slide_slide_views.xml [hunk 6]
```diff
@@ -203,67 +203,52 @@
...
+                                <t t-if="record.image_128.raw_value">
+                                    <div class="o_kanban_image_fill position-relative w-100">
+                                        <field name="image_128" t-att-alt="record.id.value" class="h-100" widget="image" options="{'img_class': 'object-fit-cover'}"/>
+                                        <field name="channel_id" class="o_inner_image position-absolute bottom-0 end-0 bg-light" widget="image" options="{'preview_image': 'image_128', 'img_class': 'object-fit-contain'}"/>
                                     </div>
-                                </div>
-                            </div>
+                                </t>
+                                <t t-else="">
+                                    <img src="/website_slides/static/src/img/channel-training-default.jpg" class="w-100" options="{'img_class': 'object-fit-cover'}"/>
```
Comment (aab-odoo) [inline addons/website_slides/views/slide_slide_views.xml:216]: missing alt attribute here (which makes runbot yellow)

File: addons/website_slides/views/slide_slide_views.xml [hunk 7]
```diff
@@ -203,67 +203,52 @@
...
+                                <t t-if="record.image_128.raw_value">
+                                    <div class="o_kanban_image_fill position-relative w-100">
+                                        <field name="image_128" class="h-100" widget="image" options="{'img_class': 'object-fit-cover'}"/>
+                                        <field name="channel_id" class="o_website_slides_inner_image position-absolute bottom-0 end-0 bg-light" widget="image" options="{'preview_image': 'image_128', 'img_class': 'object-fit-contain'}"/>
                                     </div>
-                                </div>
-                            </div>
+                                </t>
+                                <t t-else="">
+                                    <img src="/website_slides/static/src/img/channel-training-default.jpg" class="w-100" options="{'img_class': 'object-fit-cover'}" t-att-alt="record.id.value"/>
```
Comment (aab-odoo) [inline addons/website_slides/views/slide_slide_views.xml:216]: this alt doesn't really make sense does it? This is the fallback/default image which is static, so I would have expected a static string describing the image instead

**Review by aab-odoo — APPROVED** [review]
Besides the last remark on the alt attribute, I think this one is ready (to be reviewed by the website team).

File: addons/website_event_track/views/event_track_stage_views.xml [hunk 8]
```diff
@@ -77,15 +77,9 @@
...
-                <field name="fold"/>
                 <templates>
-                    <t t-name="kanban-box">
-                        <div t-attf-class="oe_kanban_global_click">
-                            <div>
-                                <strong class="o_kanban_record_title"><field name="name"/></strong>
-                            </div>
-                        </div>
+                    <t t-name="kanban-card">
+                        <field name="name" class="fw-bolder fs-5"/>
```
Comment (cth-odoo) [inline addons/website_event_track/views/event_track_stage_views.xml:82]: fw-medium is enough.

**Pull request closed without merging**